src/main/resources/lang/en_US.json

```diff
@@ -169,15 +170,13 @@
     "showSwordKills": "Show Sword Kills",
     "compactTabList": "Compact Tab List",
     "highlightSpecialEnchantments": "Highlight Special Enchantments",
-    "disableSpiritSceptreMessages": "Disable Ability Messages",
-    "showSpiritSceptreDisplay":  "Ability damage display",
+    "disableSpiritSceptreMessages": "Disable Damage Chat Messages",
+    "showSpiritSceptreDisplay": "Spirit Sceptre/Wither Blade Damage Display",
```

Comment:
What does this look like and why is it only for these two weapons?

Reply:
It looks like the last screenshot in the comment above with a bunch of screenshots. It's only for these two because they show their damage done using a specially formatted chat message.